contracts/src/v0.1/DataFeedRouter.sol

```diff
@@ -0,0 +1,242 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./interfaces/IAggregatorProxy.sol";
+
+contract DataFeedRouter is Ownable {
+    mapping(string => address) public aggregatorProxies;
```

Comment:
`aggregatorProxies` is public but it will be hard for people to explore supported proxies. How about we add a list of strings holding price feed pairs to keep track of what is currently held in this router?

Reply:
It'd be nice to have, but I think it would be enough to check through the website or by calling mapping value. I personally think it's best not to keep iterable arrays in the contract storage

Reply:
Okay! Let's keep it as it is 🙌